src/database-fs.ts

```diff
@@ -20,26 +20,33 @@ export async function clear (): Promise<void> {
 
 export async function count (): Promise<number> {
   let n = 0
-  for await (const _entry of Deno.readDir(dataFolder)) {
-    n++
+  for await (const entry of Deno.readDir(dataFolder)) {
+    // Explicit check to satisfy strict-boolean-expressions lint rule
+    if (entry.isFile === true) {
+      n++
+    }
   }
   return n
 }
 
 export async function * iterKeys (): AsyncGenerator<string> {
   for await (const entry of Deno.readDir(dataFolder)) {
     // Skip subfolders and symlinks.
-    if (entry.isFile) {
+    if (entry.isFile === true) {
       yield entry.name
     }
   }
 }
 
-export async function readData (key: string): Promise<Uint8Array | string | void> {
-  // Necessary here to thwart path traversal attacks.
+export async function readData (key: string): Promise<Uint8Array | string | undefined> {
   checkKey(key)
-  return await Deno.readFile(path.join(dataFolder, key))
-    .catch(_err => undefined)
+  try {
+    // assign to a variable before returning to satisfy no-return-await
```

Comment:
Maybe @taoeffect will have a different take, but I'm not a big fan of comments such as these, unless you're doing something weird and/or they're used sparingly. The reason is that to keep the signal-to-noise ratio high, since referring to a specific lint rule doesn't help understand the code much.

In this case, I would find it more useful, if you wanted to document why you made this change, if you'd add a GitHub comment on your PR.

Reply:
I reverted this in the latest commits.